docs/operating-systems/course-tools.md

```diff
@@ -267,4 +283,4 @@ $ sudo apt-get install docker-ce docker-ce-cli containerd.io docker-compose-plug
 C and gcc should be installed by default in Linux distributors
 
 ### Go
-In order to download [Go](https://go.dev/doc/install) on Linux click this link https://go.dev/dl/  and click the LINUX button that is underneath the feature downloads header. Once that is downloaded follow the directions that it gives you in order to complete the download
\ No newline at end of file
+In order to download [Go](https://go.dev/doc/install) on Linux click this link https://go.dev/dl/  and click the LINUX button that is underneath the feature downloads header. Once that is downloaded follow the directions that it gives you in order to complete the download
```

Comment:
Correct the link so that the user can click on it automatically as well as hide it behind a word like is done higher in the file. Also, add a period at the end of the line.